wordpress-mobile/WordPress-iOS
# Accessibility in Reader, part I: Manage cell

First PR in #8667 

In the spirit of keeping PRs as small as possible, here is the first one.

At the moment, the label "Manage" and the cell that contains it are recognised as separate components.

In this PR:
- `ReaderFollowedSitesStreamHeader` identifies itself as an accessibility element
- It also provides a label, hint, and makes the button trait explicit.
- Unit tests covering those accessibility properties.

I have declared a protocol named `Accessible`. It only declares one method now (`prepareForVoiceOver`) but the intent is to add more accessibility concerns as I go. The intent is also to make all ui elements that need to be accessible implement this protocol in extensions (like I did in this particular PR). I am not sure that's necessary or even useful, but I would like to play with the idea and see where it takes me. 

In other words, that protocol may or may not disappear in the next PR. 😊 

To test:
- Activate VoiceOver
- Navigate to the Reader tab
- Tap the "Manage" header

Expected behaviour: VoiceOver should provide a label ("Manage"), identify the header as a button, and last but not least, provide a hint ("Tapping lets you manage the sites you follow.")
